independent-projects/tools/registry-client/src/main/java/io/quarkus/registry/catalog/PlatformCatalog.java

```diff
@@ -10,6 +10,9 @@
 
     Map<String, Object> getMetadata();
 
+    @JsonIgnore
+    Platform getPlatform(String platformId);
```

Comment:
Could you do the same for this interface?

Reply:
Sorry, missed that one. Fixing now.